nemo/collections/nlp/modules/common/megatron/language_model.py

```diff
@@ -563,7 +587,16 @@ def __init__(
             if rotary_percentage < 1:
                 rotary_dim = int(rotary_dim * rotary_percentage)
             self.rotary_pos_emb = RotaryEmbedding(
-                rotary_dim, seq_len_interpolation_factor=seq_len_interpolation_factor
+                rotary_dim,
+                seq_len_interpolation_factor=seq_len_interpolation_factor,
+                # base=base,
+                # pretrain_max_positional_embeddings=pretrain_max_positional_embeddings,
```

Comment:
we can pass `max_position_embeddings` from line 546 as `pretrain_max_positional_embeddings`.